Backend/utils/db_utils.py

```diff
@@ -0,0 +1,224 @@
+import json
+import boto3
+import MentiiLogging as MentiiLogging
+from boto3.dynamodb.conditions import Key, Attr
+from botocore.exceptions import ClientError
+
+logger = MentiiLogging.getLogger()
+
+def createTableFromFile(settings_file, dbInstance):
+  try:
+    with open(settings_file) as f:
+      data = f.read()
+      settings = json.loads(data)
+
+      attribute_definitions = settings.get("AttributeDefinitions")
+      key_schema = settings.get("KeySchema")
+      provisioned_throughput = settings.get("ProvisionedThroughput")
+      table_name = settings.get("TableName")
+
+      # check for required parameters
+      if table_name is not None and key_schema is not None and attribute_definitions is not None and provisioned_throughput is not None:
+        if len(key_schema) > 0 and len(attribute_definitions) > 0:
+          table = dbInstance.create_table(
+              AttributeDefinitions=attribute_definitions,
+              KeySchema=key_schema,
+              ProvisionedThroughput=provisioned_throughput,
+              TableName=table_name
+            )
+          f.close()
+          return table
+      f.close()
+      raise(IOError)
+  except IOError as e:
+    message = "Unable to create table from file"
+    logger.exception(message + ': ' + settings_file + '\n' + str(e))
+    return message
+
+def createTableFromJson(settings_json, dbInstance):
+  if (type(settings_json) == str):
+    settings = json.loads(settings_json)
+  else:
+    settings = settings_json
+
+  attribute_definitions = settings.get("AttributeDefinitions")
+  key_schema = settings.get("KeySchema")
+  provisioned_throughput = settings.get("ProvisionedThroughput")
+  table_name = settings.get("TableName")
+
+  # check for required parameters
+  if table_name is not None and key_schema is not None and attribute_definitions is not None and provisioned_throughput is not None:
+    if len(key_schema) > 0 and len(attribute_definitions) > 0:
+      table = dbInstance.create_table(
+          AttributeDefinitions=attribute_definitions,
+          KeySchema=key_schema,
+          ProvisionedThroughput=provisioned_throughput,
+          TableName=table_name
+        )
+      return table
+  message = "Unable to create table from json"
+  logger.error(message)
+  return message
```

Comment:
Same thing  here. It's better to return an empty dictionary or none or to refactor the code.